src/sections/create-dataset/CreateDataset.tsx

```diff
@@ -42,15 +45,20 @@ export function CreateDataset({
 
   return (
     <article>
+      <NotImplementedModal show={isModalOpen} handleClose={hideModal} />
```

Comment:
Should live outside the article element, you can use a react fragment like this:
```
return (
  <>
    <NotImplementedModal show={isModalOpen} handleClose={hideModal} />
    <article>
    .....
    </article>
  <>
)
```